### PR TITLE
Reformat long URLs in NXmx documentation

### LINKED ISSUE #882: Some links are not complete as hyperlinks
Hello,

Just found these links didn't fully function on this [page](https://manual.nexusformat.org/classes/applications/NXmx.html). An extra space seems to be the cause of the problem.

https://github.com/nexusformat/definitions/blob/96ebd852e03016490e9ab4087ce02ad683945a54/applications/NXmx.nxdl.xml#L193-L197

https://github.com/nexusformat/definitions/blob/96ebd852e03016490e9ab4087ce02ad683945a54/applications/NXmx.nxdl.xml#L860-L863

### PR DESCRIPTION
**Pull request by prjemian**
Reformat long URLs in NXmx documentation to fix #882.

Also added a comment to keep URLs all on one line.  In this case, there was cosmetic line-breaking that caused the URLs to not function properly in the published documentation.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# :file: .travis.yml
```
Comment (PeterC-DLS) [inline .travis.yml:1]: Accidental re-add?

File: applications/NXmx.nxdl.xml [hunk 2]
```diff
@@ -859,7 +860,7 @@
 					<doc>
 						Name of source.  Consistency with the naming in
 						http://mmcif.wwpdb.org/dictionaries/mmcif_pdbx_v50.dic/Items
-						/_diffrn_source.pdbx_synchrotron_site.html controlled
```
Comment (PeterC-DLS) [inline applications/NXmx.nxdl.xml:862]: Still incorrect as wrapped

**Review by PeterC-DLS — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (prjemian) [reply .travis.yml:1]: Correct.  Was in my local directory as uncommitted content.  Removing.

**Comment by prjemian** [comment]
@PeterC-DLS Thanks for the review.

**Review by PeterC-DLS — APPROVED** [review]

**Pull request merged**